README.md

```diff
@@ -479,6 +479,8 @@ webpack:
        sass:
            filename: '[name].css'
            all_chunks: true
+       include_paths:
```

Comment:
I think you're missing indentations here.

`include_paths` is a child option of `sass` right?